src/test/java/org/jclouds/vcloud/director/v1_5/compute/strategy/VCloudDirectorComputeServiceContextLiveTest.java

```diff
@@ -64,19 +63,27 @@ public void testLaunchClusterWithMinDisk() throws RunNodesException {
 
       TemplateBuilder templateBuilder = context.getComputeService().templateBuilder();
 
-      Template template = templateBuilder.imageNameMatches("centos6.4x64") // TAI
-                                         .build();
+      Template template = templateBuilder
+              .locationId("https://emea01.canopy-cloud.com/api/vdc/e931a09d-131f-4aaa-a667-efbe02eba428") // TAI2.0
+              //.imageNameMatches("centos6.4x64") // TAI
+              .imageNameMatches("CentOS_66_x64_platform") // TAI2.0
+              .build();
       // test passing custom options
       VCloudDirectorTemplateOptions options = template.getOptions().as(VCloudDirectorTemplateOptions.class);
 
-      Set<? extends NodeMetadata> nodes = context.getComputeService().createNodesInGroup(name, numNodes, template);
-      assertEquals(numNodes, nodes.size(), "wrong number of nodes");
-      for (NodeMetadata node : nodes) {
+      options//.networks("Deployment_Network_01"); // TAI
+              .networks("Operational_Network_01"); // TAI2.0
+
+      NodeMetadata node = null;
+      try {
+         Set<? extends NodeMetadata> nodes = context.getComputeService().createNodesInGroup(name, 1, template);
+         node = Iterables.getOnlyElement(nodes);
          logger.debug("Created Node: %s", node);
          SshClient client = context.utils().sshForNode().apply(node);
          client.connect();
          ExecResponse hello = client.exec("mount");
          logger.debug(hello.getOutput().trim());
+      } finally {
          context.getComputeService().destroyNode(node.getId());
```

Comment:
Guard  against `node == null`, or move the `nodes = ...` and `node = ...` outside of the try block.